lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -418,7 +418,7 @@ def collapse_columns(df: pd.DataFrame, stub: str, pid='enrollid') -> pd.DataFram
     column called "Race". Removes the original "Race*" option columns. Returns
     the new DataFrame.
     """
-    stub_data = df.filter(regex=f'{pid}|{stub}*', axis='columns')
+    stub_data = df.filter(regex=f'{pid}|{stub}.*', axis='columns')
```

Comment:
Good bug find/fix! I think it would be a great improvement to add a doctest here as well if possible.

Reply:
Good suggestion, I need to get in the habit of that! Added and ready for review.